### PR TITLE
Default time support

### PR DESCRIPTION
**Pull request by jdgblinq**
#25 default time functionality.

### CODE REVIEW

File: datetimepicker.js [hunk 1]
```diff
@@ -130,12 +136,62 @@ angular.module('ui.bootstrap.datetimepicker',
             };
           }
         ],
-        link: function(scope) {
+        link: function(scope, element, attrs) {
+          if (typeof scope.last_date === 'undefined') { scope.last_date = [] }
+          if (typeof(scope.last_date[scope.$id]) === 'undefined') { scope.last_date[scope.$id] = null; }
+          function sameDate(date1, date2) {
```
Comment (jdgblinq) [inline datetimepicker.js]: I just realized this function is unused and can be removed.

File: example.js [hunk 2]
```diff
@@ -2,7 +2,8 @@ angular.module('plunker', ['ui.bootstrap', 'ui.bootstrap.datetimepicker'])
 
 var DateTimePickerDemoCtrl = function ($scope, $timeout) {
   $scope.dateTimeNow = function() {
-    $scope.date = new Date();
+    $scope.date = null;
+    $scope.date1 = new Date();
```
Comment (zhaber) [inline example.js]: Please revert

Comment (jdgblinq) [inline example.js]: Revert what?

Comment (jdgblinq) [inline example.js]: Are you asking me to close the PR?

**Pull request closed without merging**